processor/src/chiplets/aux_trace/mod.rs

```diff
@@ -466,13 +468,34 @@ fn build_mstream_request<E: FieldElement<BaseField = Felt>>(
     factor1 * factor2
 }
 
+/// Builds `RCOMBBASE` requests made to the memory chiplet.
+fn build_rcomb_base_request<E: FieldElement<BaseField = Felt>>(
+    main_trace: &MainTrace,
+    alphas: &[E],
+    i: usize,
+) -> E {
+    let tz0 = main_trace.helper(0, i);
+    let tz1 = main_trace.helper(1, i);
+    let tzg0 = main_trace.helper(2, i);
+    let tzg1 = main_trace.helper(3, i);
+    let a0 = main_trace.helper(4, i);
+    let a1 = main_trace.helper(5, i);
+    let z_ptr = main_trace.stack_element(13, i);
+    let a_ptr = main_trace.stack_element(14, i);
+
+    let factor1 = main_trace.compute_memory_read_request(alphas, i, z_ptr, [tz0, tz1, tzg0, tzg1]);
+    let factor2 = main_trace.compute_memory_read_request(alphas, i, a_ptr, [a0, a1, ZERO, ZERO]);
```

Comment:
Given previous comment, this would call the new `build_mem_word_request()`

Reply:
Makes sense, changed!